.github/workflows/actions.yaml

```diff
@@ -128,7 +128,7 @@ jobs:
       - name: Change goversioninfo in jjversion.go
         run: sed -i "s/\/\/go:generate goversioninfo/\/\/go:generate goversioninfo -file-version=${{ env.VERSION }} -product-version=${{ env.VERSION }} -ver-major=${{ env.MAJOR }} -ver-minor=${{ env.MINOR }} -ver-patch=${{ env.PATCH }} -ver-build=0 -product-ver-major=${{ env.MAJOR }} -product-ver-minor=${{ env.MINOR }} -product-ver-patch=${{ env.PATCH }} -product-ver-build=0/g" jjversion.go
       - run: echo $PATH
-      - run: go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo
+      - run: go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@latest
```

Comment:
This change resolves an issue that occurred in the build of the previous commit where this package could not be installed:

Run go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo
  go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo
  shell: /usr/bin/bash -e {0}
  env:
    GOROOT: /opt/hostedtoolcache/go/1.18.0/x64
    VERSION: 0.5.[3](https://github.com/jjliggett/jjversion/runs/5965703033?check_suite_focus=true#step:19:3)
    MAJOR: 0
    MINOR: [5](https://github.com/jjliggett/jjversion/runs/5965703033?check_suite_focus=true#step:19:5)
    PATCH: 3
no required module provides package github.com/josephspurrier/goversioninfo/cmd/goversioninfo; to add it:
	go get github.com/josephspurrier/goversioninfo/cmd/goversioninfo
Error: Process completed with exit code 1.

( https://github.com/jjliggett/jjversion/runs/5965703033?check_suite_focus=true )

This was recommended in the following comment: https://github.com/josephspurrier/goversioninfo/issues/54#issuecomment-1004969187